python/paddle/fluid/tests/unittests/test_executor_return_tensor_not_overwriting.py

```diff
@@ -0,0 +1,116 @@
+#  Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+import unittest
+import numpy as np
+import paddle.fluid.core as core
+import paddle.fluid as fluid
+from op_test import OpTest
+
+
+class TestExecutorReturnTensorNotOverwritingWithOptest(OpTest):
+    def setUp(self):
+        pass
+
+    def calc_add_out(self, place=None, parallel=None):
+        self.x = np.random.random((2, 5)).astype(np.float32)
+        self.y = np.random.random((2, 5)).astype(np.float32)
+        self.out = np.add(self.x, self.y)
+        self.inputs = {
+            'X': OpTest.np_dtype_to_fluid_dtype(self.x),
+            'Y': OpTest.np_dtype_to_fluid_dtype(self.y)
+        }
+        self.outputs = {'Out': self.out}
+        self.op_type = "elementwise_add"
+        self.dtype = np.float32
+        outs, fetch_list = self._calc_output(place, parallel=parallel)
+        return outs
+
+    def calc_mul_out(self, place=None, parallel=None):
+        self.x = np.random.random((2, 5)).astype(np.float32)
+        self.y = np.random.random((5, 2)).astype(np.float32)
+        self.out = np.dot(self.x, self.y)
+        self.inputs = {
+            'X': OpTest.np_dtype_to_fluid_dtype(self.x),
+            'Y': OpTest.np_dtype_to_fluid_dtype(self.y)
+        }
+        self.outputs = {'Out': self.out}
+        self.op_type = "elementwise_mul"
+        self.dtype = np.float32
+        outs, fetch_list = self._calc_output(place, parallel=parallel)
+        return outs
+
+    def test_executor_run_twice(self):
+        places = [fluid.CPUPlace()]
+        if fluid.is_compiled_with_cuda() and fluid.core.op_support_gpu(
```

Comment:
Remove `op_support_gpu`. It is not necessary. Leaving `fluid.is_compiled_with_cuda()` is ok.

Reply:
Done, thanks.